Example/WalletApp/PresentationLayer/Wallet/Welcome/WelcomePresenter.swift

```diff
@@ -16,9 +16,11 @@ final class WelcomePresenter: ObservableObject {
     }
     
     func onGetStarted() {
-        let pasteboard = UIPasteboard.general
-        let clientId = try? Networking.interactor.getClientId()
-        pasteboard.string = clientId
+        
+        // Commenting this out as it breaks UI tests that copy/paste URI
+//        let pasteboard = UIPasteboard.general
+//        let clientId = try? Networking.interactor.getClientId()
+//        pasteboard.string = clientId
```

Comment:
Question for reviewers...
Not sure how much this is used and whether I should keep it and devise a workaround.

Reply:
or any other way you like but we need to be able to get it. It's sometimes important to check the relay logs for particular client id.